pom.xml

```diff
@@ -215,6 +218,71 @@
                         </execution>
                     </executions>
                 </plugin>
+                <plugin>
+                    <artifactId>maven-dependency-plugin</artifactId>
+                    <version>2.10</version>
+                    <executions>
+                        <execution>
+                            <id>unpack</id>
+                            <phase>generate-sources</phase>
+                            <goals>
+                                <goal>unpack</goal>
+                            </goals>
+                            <configuration>
+                                <artifactItems>
+                                    <artifactItem>
+                                        <!-- includes common copyright headers for sources -->
+                                        <groupId>org.kaazing</groupId>
+                                        <artifactId>community.license</artifactId>
+                                        <type>jar</type>
+                                    </artifactItem>
+                                </artifactItems>
+                                <overWriteReleases>true</overWriteReleases>
+                                <overWriteSnapshots>true</overWriteSnapshots>
+                                <failOnMissingClassifierArtifact>true</failOnMissingClassifierArtifact>
+                            </configuration>
+                        </execution>
+                    </executions>
+                </plugin>
+                <plugin>
+                    <groupId>com.mycila</groupId>
+                    <artifactId>license-maven-plugin</artifactId>
+                    <version>2.3</version>
+                    <configuration>
+                        <header>${project.build.directory}/dependency/COPYRIGHT-community</header>
+                        <failIfMissing>true</failIfMissing>
+                        <useDefaultExcludes>false</useDefaultExcludes>
+                        <includes>
+                            <include>src/**</include>
+                        </includes>
+                        <excludes>
+                            <exclude>**/*.png</exclude>
```

Comment:
It does give "[WARNING] Unknown file extension:".  I think unless we want to list every single type of file extension (or no extension), which I think is a bad idea, this kind of warning is a "necessary evil".  Let me know if you think otherwise.